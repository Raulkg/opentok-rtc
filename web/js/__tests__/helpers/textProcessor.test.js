var textProcessor = require('../../helpers/textProcessor.js');
test('snapshot for textProcessor component', () => {
  expect(textProcessor).toMatchSnapshot();
});