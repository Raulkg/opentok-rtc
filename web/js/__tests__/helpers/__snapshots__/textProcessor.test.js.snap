// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for textProcessor component 1`] = `
Object {
  "TextProcessor": Object {
    "TYPE": Object {
      "TXT": "txt",
      "URL": "url",
    },
    "parse": [Function],
  },
}
`;
